scripts/ci/detect_bookrunner_failures.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# This script checks that the number of failures in a bookrunner run are below
+# the threshold declared below.
+#
+# The threshold is roughly computed as: `1.05 * <number of expected failures>`
+# The extra 5% allows us to account for occasional timeouts. It is reviewed and
+# updated whenever the Rust toolchain version is updated.
+THRESHOLD=652
```

Comment:
nit: can we add the calculation here instead of relying on the person that is updating this?
```bash
EXPECTED=<num>
THRESHOLD=$(expr ${EXPECTED} \* 105 / 100) # Add 5% threshold.
```

Reply:
Done!